profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyContactDisplay.html

```diff
@@ -26,13 +26,13 @@
 <body>
 <wicket:panel>
 
-	<div class="mainSection editable" tabindex=0>
+	<div class="mainSection editable">
 		<div class="mainSectionHeading">
-            <i class="si si-edit"></i>
 			<span wicket:id="heading">[Contact Information]</span>
 		</div>
 
 		<a href="#" wicket:id="editButton" class="edit-button" role="button">
+			<i class="si si-edit"></i>
```

Comment:
This was the incorrectly placed icon. It has it here in prod, minor but a lot more noticeable after this change.

<img src=https://github.com/sakaiproject/sakai/assets/27447/439ff360-a6e9-4b62-9e05-7a3bf8271b8f width=400px>